Assets/Scripts/Games/NipInTheBud/NipInTheBud.cs

```diff
@@ -89,20 +109,34 @@ public class NipInTheBud : Minigame
     {
         public static NipInTheBud instance;
```

Comment:
it'd be nice if we could set a standard of never using static instances when unneeded, just use eventCaller.gameManager.TryGetMinigame() in the events list or gameManager.TryGetMinigame() in the minigame script